src/System.IO.FileSystem.AccessControl/tests/Configurations.props

```diff
@@ -1,7 +1,9 @@
 ﻿<Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard-Windows_NT;
+      netcoreapp-Windows_NT;
+      netfx-Windows_NT;
```

Comment:
Nit: netfx and uap only build on windows.

Reply:
same comment as below.